### PR TITLE
Add limix-plot

### PR DESCRIPTION
**Pull request by horta**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by horta** [comment]
Hi @conda-forge/help-python , it can't download the source in neither of the following urls:

- https://pypi.io/packages/source/l/limix-plot/limix-plot-0.1.0.tar.gz
- https://pypi.org/packages/source/l/limix-plot/limix-plot-0.1.0.tar.gz
- https://files.pythonhosted.org/packages/source/l/limix-plot/limix-plot-0.1.0.tar.gz

**Comment by horta** [comment]
Sorry, I think this is being handled here: https://github.com/pypa/warehouse/issues/5579

File: recipes/limix-plot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d9a73d36eed7cc3df893a5923d404410f06b08adf6438a1398b69d25bff86df8
+
+build:
+  skip: True  # [py27]
```
Comment (marcelotrevisani) [inline recipes/limix-plot/meta.yaml]: Please remove this skip add ``noarch: python``
In host and run requirements please add
```
  - python > 3
```

File: recipes/limix-plot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  sha256: d9a73d36eed7cc3df893a5923d404410f06b08adf6438a1398b69d25bff86df8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python > 3
```
Comment (chrisburr) [inline recipes/limix-plot/meta.yaml]: I'm not sure if the space after the `>` works everywhere in `conda-build`:
```suggestion
    - python >3
```

File: recipes/limix-plot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - setuptools
+    - pytest-runner
+  run:
+    - adjustText >=0.7.3
+    - Pillow >=6.0.0
+    - matplotlib >=3.0.3
+    - numpy >=1.16.2
+    - pandas >=0.24.2
+    - pytest >=4.4.1
+    - python > 3
```
Comment (chrisburr) [inline recipes/limix-plot/meta.yaml]: Same here:
```suggestion
    - python >3
```

**Comment by horta** [comment]
Sorry about those files. Missed them completely!

**Comment by chrisburr** [comment]
If the macOS failure is from matplotlib crashing we could ignore it and merge anyway as this is `noarch` and will only be built on Linux.

**Comment by horta** [comment]
@chrisburr thx ! I will only give another try...

**Comment by horta** [comment]
@chrisburr thx ! I think it is now all working =)

**Pull request merged**